app/components/SelectedTranscript.tsx

```diff
@@ -21,19 +21,35 @@ export const SelectedTranscript: React.FC<SelectedTranscriptProps> = ({
         {latestDoc ? latestDoc.title : "Meeting title"}
       </h1>
       <h2 className="mt-8 mb-5">Attendees</h2>
-      <p>{latestDoc ? latestDoc.attendees : "Meeting attendees"}</p>
-      <h2 className="mt-8 mb-5">key discussion points</h2>
-      {latestDoc
-        ? discussionPoints.map((item: string, index: number) => (
+      {latestDoc && latestDoc.attendees ? (
```

Comment:
I like you use of ternaries here. Makes the code very readable. The html here is quite repetitive, this seems a good opportunity for a more generic unordered list component to make it even more clear.